### PR TITLE
Add Get-BitlockerInformation

### PR DESCRIPTION
**Pull request by andrewbaker-uk**

### CODE REVIEW

File: PowerShell Scanners/Get Bitlocker information/Scan Profile.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<AdminArsenal.Export Code="PDQInventory" Name="PDQ Inventory" Version="19.0.40.0" MinimumVersion="19.0">
+  <ScanProfile>
+    <Collections type="list" />
+    <Scanners type="list">
+      <Scanner>
+        <ModifiedDate>2020-05-04T11:19:45.0000000+01:00</ModifiedDate>
+        <Name>Bitlocker Information</Name>
+        <Script>if (-Not(Get-Command Get-BitlockerVolume -ErrorAction SilentlyContinue)) {
```
Comment (bryan-pdq) [inline PowerShell Scanners/Get Bitlocker information/Scan Profile.xml]: This scanner export isn't referencing the .ps1 file that your added. Can you change this so it uses the FileName field and references the file on the C drive?

Comment (andrewbaker-uk) [reply PowerShell Scanners/Get Bitlocker information/Scan Profile.xml]: Hi Bryan, I've made the suggested changes to the scan profile.

**Review by andrewbaker-uk — COMMENTED** [review]
Made suggested changes. Requested review

**Comment by steviecoaster** [comment]
I think we should revert 43fdec4. The `PS - ` is used in the Scan Profile name such that it's easy to find in the list of Scanners, but for consistency with the rest of the Scanners here that should be dropped from the actual `.ps1` filename. IMHO, maintainers may see otherwise.

**Comment by Colby-PDQ** [comment]
> 
> 
> I think we should revert [43fdec4](https://github.com/pdq/PowerShell-Scanners/commit/43fdec482d9bfcbc54a3f710affd4d5d3d4d2af9). The `PS - ` is used in the Scan Profile name such that it's easy to find in the list of Scanners, but for consistency with the rest of the Scanners here that should be dropped from the actual `.ps1` filename. IMHO, maintainers may see otherwise.

steviecoaster is correct. Please remove `PS - ` from the name of your .ps1 file. https://github.com/pdq/PowerShell-Scanners/blob/master/Policies/Contributing.md#folder-structure

**Comment by andrewbaker-uk** [comment]
> > I think we should revert [43fdec4](https://github.com/pdq/PowerShell-Scanners/commit/43fdec482d9bfcbc54a3f710affd4d5d3d4d2af9). The `PS - ` is used in the Scan Profile name such that it's easy to find in the list of Scanners, but for consistency with the rest of the Scanners here that should be dropped from the actual `.ps1` filename. IMHO, maintainers may see otherwise.
> 
> steviecoaster is correct. Please remove `PS - ` from the name of your .ps1 file. https://github.com/pdq/PowerShell-Scanners/blob/master/Policies/Contributing.md#folder-structure

Opps, consider it reverted

**Comment by Colby-PDQ** [comment]
@theterminator3000 Something weird happened. Did you delete your commits? They're no longer showing up on this PR, and I don't see anything on your fork.

**Comment by andrewbaker-uk** [comment]
Weird, I haven't done anything this end

**Pull request merged**